concent-builder/consts.yml

```diff
@@ -1,5 +1,8 @@
-shared_user:       builder
-data_dir:          /var/concent-data
-build_dir:         /home/{{ shared_user }}/build/{{ cluster }}
-local_secret_dir:  ../../concent-secrets
-deployment_values: ../../concent-deployment-values
+shared_user:                   builder
+data_dir:                      /var/concent-data
+build_dir:                     /home/{{ shared_user }}/build/{{ cluster }}
+local_secret_dir:              ../../concent-secrets
+deployment_values:             ../../concent-deployment-values
+deployment_values_repository:  https://github.com/golemfactory/concent-deployment-values.git
```

Comment:
Why are you not taking this address from our file that lists repositories?

Reply:
This is value of `concent-deployment-values` repository which is not included in our lists of repository.